### PR TITLE
[EN-764] Update kafkautils for stream server

### PR DESCRIPTION
**Pull request by cnkarz**
- Update utils for stream server changes
- Tests

### CODE REVIEW

File: kafkautils/topic.go [hunk 1]
```diff
@@ -6,20 +6,24 @@ import (
...
 
-var TopicDelimiter = "."
-
 type env string
 
 const (
 	prod    env = "prod"
 	staging     = "staging"
 	dev         = "dev"
-	test        = "test"
```
Comment (azhang) [inline kafkautils/topic.go:22]: Let's include test

File: kafkautils/topic.go [hunk 2]
```diff
@@ -32,71 +36,41 @@ const (
...
-}
-
-func NewTopic(en, ty, author, connectorName string, version *semver.Version, msg proto.Message) Topic {
-	return Topic{
-		Env:           env(en),
-		MsgType:       msgType(ty),
-		Author:        author,
-		ConnectorName: connectorName,
-		Version:       version,
-		EventName:     strings.ReplaceAll(strings.ToLower(string(msg.ProtoReflect().Descriptor().FullName())), ".", "_"),
```
Comment (azhang) [inline kafkautils/topic.go:66]: Did you intend to remove this? This generates the eventname directly from the proto.Message

File: kafkautils/topic.go [hunk 3]
```diff
@@ -32,71 +36,41 @@ const (
 )
 
 type Topic struct {
-	Env           env
-	MsgType       msgType
-	Author        string
-	ConnectorName string
-	Version       *semver.Version
```
Comment (azhang) [inline kafkautils/topic.go:39]: I think we should keep topic as Semver

**Comment by azhang** [comment]
I think there were some changes in this kafkautils module that might've been overwritten inadvertently

File: kafkautils/topic.go (hunk 2 above)
Comment (cnkarz) [reply kafkautils/topic.go]: no 😅

**Comment by cnkarz** [comment]
I thought it was the same as in blep repo so I just copy-pasted from there 😄 I reverted the changes

**Comment by azhang** [comment]
Looking at the diffs between this and kafkautils in monorepo, why is context removed from Listen?

**Comment by cnkarz** [comment]
> Looking at the diffs between this and kafkautils in monorepo, why is context removed from Listen?

It was added with protoregistry changes but it doesn't hurt to leave it there I guess 😅 I put it back

**Comment by cnkarz** [comment]
Closed due to being outdated and replaced with #4

**Pull request closed without merging**